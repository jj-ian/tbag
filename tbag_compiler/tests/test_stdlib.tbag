true {
	strPrintLine("hi");
	intPrintLine(666);
	boolPrintLine(true);
	boolPrintLine(false);
	endgame;
}

#import stdlib
