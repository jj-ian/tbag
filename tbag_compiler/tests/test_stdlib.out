hi
666
true
false
